integration-tests/tests/test_wallet_rpc.py

```diff
@@ -0,0 +1,69 @@
+import random
+import pytest
+import jsonschema
+import json
+from conftest import option, user_1, user_2
+from test_cases import RpcTestCase, TransactionTestCase
+
+
+@pytest.mark.wallet
+@pytest.mark.tx
+class TestTransactionRpc(TransactionTestCase):
+
+    @pytest.mark.parametrize(
+        "method, params",
+        [
+            (
+                "wallet_checkRecentHistoryForChainIDs",
+                [[31337], ["0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"]],
+            ),
+            (
+                "wallet_getPendingTransactionsForIdentities",
+                [[{"chainId": None, "hash": None}]],
+            ),
+        ],
+    )
+    def test_tx_(self, method, params):
+
+        _id = str(random.randint(1, 9999))
+
+        if method in ["wallet_getPendingTransactionsForIdentities"]:
+            params[0][0]["chainId"] = self.network_id
+            params[0][0]["hash"] = self.tx_hash
+
+        response = self.rpc_request(method, params, _id)
+        self.verify_is_valid_json_rpc_response(response)
+        with open(f"{option.base_dir}/schemas/{method}", "r") as schema:
+            jsonschema.validate(instance=response.json(), schema=json.load(schema))
+
+    def test_create_multi_transaction(self):
+        response = self.wallet_create_multi_transaction()
+        
+        # how to create schema:
+        # from schema_builder import CustomSchemaBuilder
+        # CustomSchemaBuilder("wallet_createMultiTransaction").create_schema(response.json())
+        
+        with open(f"{option.base_dir}/schemas/wallet_createMultiTransaction", "r") as schema:
+            jsonschema.validate(instance=response.json(), schema=json.load(schema))
```

Comment:
I'd suggest to make a separate method for this with a schema name as an input parameter, something like `def validate_json_schema(schema_name)`